packages/interop/package.json

```diff
@@ -60,7 +60,6 @@
     "@libp2p/peer-id-factory": "^3.0.3",
     "@libp2p/tcp": "^8.0.4",
     "@libp2p/websockets": "^7.0.4",
-    "aegir": "^41.0.0",
```

Comment:
This should be left in place.  Each package in a monorepo should declare all of it's dependencies, otherwise we have some deps here, some deps there, and then it becomes hard to reason about where things are coming from and why.